collectors/charts.d.plugin/speedtest/Makefile.inc

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Why do we need this file?

Reply:
i dont think we do for stuff in netdata/community

Reply:
I saw that the other collectors in community had them and kept it in - don't think we need it.